TOOLING.md

```diff
@@ -0,0 +1,15 @@
+# Tooling
+
+This documents describes at a high level some the key tools used across `near-js`.
```

Comment:
This document describes at a high level some of the key tools used across `near-js` ?

Reply:
https://github.com/near/near-api-js/pull/999#discussion_r981752847